plugins/LdapPlugin/src/opengrok/auth/plugin/LdapUserPlugin.java

```diff
@@ -47,7 +47,7 @@
     protected static final String OBJECT_CLASS = "objectclass";
     
     private String objectClass;
-    private Pattern usernameCnPattern = Pattern.compile("(cn=[a-zA-Z_]+)");
+    private final Pattern usernameCnPattern = Pattern.compile("(cn=[a-zA-Z0-9_-]+)");
 
     private boolean isAlphanumeric(String str) {
```

Comment:
I know that this change is not about this function but it should really be in StringsUtils and I think we even have this class in opengrok already

Reply:
sure